extensions/amp-ad/0.1/test/test-amp-ad-3p-impl.js

```diff
@@ -294,8 +295,6 @@ describe('amp-ad-3p-impl', () => {
         const newIntersection = ad3p.getIntersectionElementLayoutBox();
         expect(newIntersection).not.to.deep.equal(intersection);
         expect(newIntersection.top).to.equal(intersection.top + 100);
-        expect(newIntersection.width).to.equal(300);
```

Comment:
I thought `expect(newIntersection.top).to.equal(intersection.top + 100);` would be enough to test if the value is cached. added back anyway since it's not related to the pr.